crates/tests/databases-tests/src/postgres/snapshots/databases_tests__postgres__mutation_tests__negative__experimental_insert_artist_missing_column.snap

```diff
@@ -5,6 +5,6 @@ expression: result
 {
   "message": "Invalid request",
   "details": {
-    "detail": "Unable to insert into 'Artist'. Column 'ArtistId' is missing."
+    "detail": "Unable to mutate 'Artist'. Column 'ArtistId' is missing."
```

Comment:
Wishlist: It would be nice if errors could still relate directly to what the user was doing, rather than using more abstract terms.

Reply:
I made this part of the error message parameters, so now you can pass `"insert into"` or `"update"` from the caller.